paasta_itests/paasta_deployd.feature

```diff
@@ -21,27 +18,65 @@ Feature: paasta-deployd deploys apps
      Then we set a new command for our service instance to "/bin/bash -c 'echo deploy-all-the-things'"
     Given I have yelpsoa-configs for the marathon job "test-service.main"
      Then the appid for "test-service.main" should have changed
-     Then we should not see the old version listed in marathon after 60 seconds
+     Then we should not see the old version listed in marathon after 70 seconds
```

Comment:
This was just during troubleshooting, don't think it actually helped. I'll put it back to 60